third_party/move/move-vm/runtime/src/config.rs

```diff
@@ -73,11 +73,11 @@ impl Default for VMConfig {
             enable_depth_checks: true,
             optimize_trusted_code: false,
             paranoid_ref_checks: false,
-            enable_capture_option: false,
+            enable_capture_option: true,
```

Comment:
this should be set to true because enable_capture_option is by default turned on on devnet.